lib/ansible/utils/path.py

```diff
@@ -84,12 +83,11 @@ def makedirs_safe(path, mode=None):
     if not os.path.exists(b_rpath):
         try:
             if mode:
-                os.makedirs(b_rpath, mode)
+                os.makedirs(b_rpath, mode, exist_ok=True)
             else:
-                os.makedirs(b_rpath)
-        except OSError as e:
-            if e.errno != EEXIST:
-                raise AnsibleError("Unable to create local directories(%s): %s" % (to_native(rpath), to_native(e)))
+                os.makedirs(b_rpath, exist_ok=True)
+        except OSError as ex:
+            raise AnsibleError(f"Unable to create local directories: {rpath!r}") from ex
```

Comment:
Since the path is being quoted, we can drop the colon and end the message with a period:

```suggestion
            raise AnsibleError(f"Unable to create local directories {rpath!r}.") from ex
```